### PR TITLE
fix: fix sessionauth outside of router

### PR DESCRIPTION
**Pull request by porcellus**
## Summary of change

Fixes rare edge case in usage of SessionAuth outside of routers (and in completely separate components.

## Related issues

-   

## Test Plan

Added unit test

## Documentation changes

N/A, this is a fix

## Checklist for important updates

-   [x] Changelog has been updated
-   [x] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [x] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [x] If added a new recipe interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [x] If I added a new recipe, I also added the recipe entry point into the `size-limit` section of `package.json` with the size limit set to the current size rounded up.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,24 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [unreleased]
 
-### Changed
-
--   Adds unit tests to make sure that recipe functions are exported correctly
-
-## [0.24.4] - 2022-08-07
-
--   Fixes styling issue for login with github
+### Bug fixes
```
Comment (rishabhpoddar) [inline CHANGELOG.md:10]: There is already a section called Fixes for this release. Please add this comment in there.

File: CHANGELOG.md [hunk 2]
```diff
@@ -155,6 +144,21 @@ function ProtectedHomeComponent() {
...
+
+### Changed
+
+-   Updated `react-scripts` in all examples
+-   Updated READMEs for example apps, removed information that is no longer relevant.
+
+### Fixed
+
+-   Fixed capitalization of GitHub on the UI (fixes https://github.com/supertokens/supertokens-auth-react/issues/539)
+
```
Comment (rishabhpoddar) [inline CHANGELOG.md:161]: Not sure why this is showing. Is the base branch properly merged with the latest?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (porcellus) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (porcellus) [reply CHANGELOG.md:158]: This fixes a merge error from the base branch. I can commit this "fix" there directly if you prefer.

**Review by rishabhpoddar — APPROVED** [review]
